plugins/manager.go

```diff
@@ -389,14 +383,7 @@ func (m *Manager) install(t PluginType, name string, src string) ([]string, stri
 			}
 			filenames = append(filenames, yamlPath)
 		} else if fileName == name+".json" {
-			confDir, err := common.GetConfLoc()
-			if nil != err {
-				return filenames, "", err
-			}
-			err = unzipTo(file, path.Join(confDir, PluginTypes[t], fileName))
-			if err != nil {
-				return filenames, "", err
-			}
+			unzipTo(file, path.Join(m.etcDir, PluginTypes[t], fileName))
```

Comment:
Do we need to handle the error here? Shall we fail the installation if error happens?